lib/objects/objectsUtils.js

```diff
@@ -332,13 +333,13 @@ function getDefaultAdminRights(acl, _isState) {
 
 function getUserGroup(objects, user, callback) {
     if (!user || typeof user !== 'string' || !user.match(regUser)) {
-        console.log('invalid user name: ' + user);
+        console.log(`invalid user name: ${user}`);
         user = JSON.stringify(user);
         // deep copy
-        return callback.call(objects, 'invalid user name: ' + user, user, [], deepClone(defaultAcl.acl));
+        return tools.maybeCallbackWithError(callback, objects, `invalid user name: ${user}`, user, [], deepClone(defaultAcl.acl));
     }
     if (users[user]) {
-        return callback.call(objects, null, user, users[user].groups, users[user].acl);
+        return tools.maybeCallbackWithError(callback, objects, null, user, users[user].groups, users[user].acl);
```

Comment:
`objects` is the type of `this`, not the second argument to `maybeCallbackWithError` which should be an Error or null

Reply:
Thanks, crahsed the tests